src/RxTelegram.Bot/Interface/BaseTypes/ChatBoost.cs

```diff
@@ -0,0 +1,27 @@
+namespace RxTelegram.Bot.Interface.BaseTypes;
+
+/// <summary>
+/// This object contains information about a chat boost.
+/// </summary>
+public class ChatBoost
+{
+    /// <summary>
+    /// Unique identifier of the boost
+    /// </summary>
+    public string BoostId { get; set; }
+
+    /// <summary>
+    /// Point in time (Unix timestamp) when the chat was boosted
+    /// </summary>
+    public int AddDate { get; set; }
```

Comment:
Do we want to change all date types (int) to datetime? Because we can use the UnixDateTimeConverter?